contracts/hooks/examples/FullRange.sol

```diff
@@ -47,13 +48,7 @@ contract FullRange is BaseHook, ILockCallback {
 
     struct PoolInfo {
         uint128 liquidity;
```

Comment:
is this liquidity param redundant with the erc20 `totalSupply`??

Reply:
ah it is! i am thinking of getting rid of this liquidity param anyways.